encryption/modern_caesar.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <string>
+#include "i_encryption.h"
+
+namespace encryption {
+
+    class ModernCaesar: public IEncryption {
+    private:
+        int const seed;
```

Comment:
To avoid naming shadows, please, use different names for constructor arguments and class fields. you can use m_seed (more old, like in java) or _seed, or seed_ for example.